opentok/opentok.py

```diff
@@ -1021,6 +1055,11 @@ def dial(self, session_id, token, sip_uri, options=[]):
         if "secure" in options:
             payload["sip"]["secure"] = options["secure"]
 
+        if "observeForceMute" in options:
+            observeForceMute = True
+            payload["sip"]["observeForceMute"] = options["observeForceMute"]
+
+
```

Comment:
Remove extra line

Reply:
Ok, thank you! Will remove the space and fix the build issues.